### DISCUSSION TITLE
Build for MacOS?

### DISCUSSION THREAD

**Discussion by Helen0721** [facebook/CacheLib#305] [Q&A]
Hi. I'm fairly new to this code base. I wonder if it is possible to build the relevant dependencies on M1 MacOS? The build.sh file doesn't seem to provide a way. Thanks.

**Comment by therealgymmy** [comment]
Hi cachelib does not officially support macos. You will need to build it on a supported linux platform such as Ubuntu 22.04

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
